setup.cfg

```diff
@@ -147,11 +143,12 @@ tests =
     pylint-plugin-utils==0.7
     # type-checking
     mypy==0.942
-    types-requests==2.27.15
-    types-tabulate==0.8.6
-    types-toml==0.10.4
+    types-requests>=2.27.15
+    types-tabulate>=0.8.6
+    types-toml>=0.10.4
```

Comment:
We don't really care that much about specific versions of these, will reduce a bit of noise with dependabot.